cli/integration_tests/prune/produces_valid_turbo_json.t

```diff
@@ -19,6 +19,7 @@ Make sure we prune tasks that reference a pruned workspace
         "dependsOn": [],
         "inputs": [],
         "outputMode": "full",
+        "passthroughEnv": [],
```

Comment:
This shouldn't be added, since it's an experimental field 🤔 , need to figure out why it's happening